keps/sig-multicluster/1645-multi-cluster-services-api/README.md

```diff
@@ -1026,6 +1032,18 @@ Session affinity affects a service as a whole for a given consumer. The derived
 service's session affinity will be decided according to the conflict resolution
 policy.
 
+#### Internal Traffic Policy
+
+Internal traffic policy affects a service as a whole for a given consumer. The derived
```

Comment:
Could you please clarify that how consumers define traffic distribution within the multi-clusters?

Are we going to reuse the existing Internal Traffic Policy and TrafficDistribution in the service? I feel it's confusing as these fields define how the traffic to be routed within the cluster itself, not about multi-clusters env.

Reply:
Will try to find if there are places where we say that but yes this is from the Service field and PreferSameAZ could works across clusters (different clusters using the same AZ) and if not it's still valuable to prefer something in the local cluster and then fallback to the rest of the endpoints whether they are in a different clusters or not (which would be true for preferring the same host also)